pkg/ottl/contexts/internal/span.go

```diff
@@ -585,3 +587,17 @@ func accessStatusMessage[K SpanContext]() ottl.StandardGetSetter[K] {
 		},
 	}
 }
+
+func accessFlags[K SpanContext]() ottl.StandardGetSetter[K] {
```

Comment:
please also add a test case for accessing flags to the [unit test of the SpanPathGetSetter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/80fc977f9bf264b312284b9bf08cb095480d5ce2/pkg/ottl/contexts/internal/span_test.go#L27)